frontend/src/components/homepage/homepage.tsx

```diff
@@ -1,6 +1,6 @@
-import Timer from "./timer";
+import Timer from "./Timer";
```

Comment:
Naming is wrong since the file is called timer.tsx causing failed to resolve import
![Screenshot 2024-05-23 at 7 17 09 PM](https://github.com/Thangnguyenvn98/GatheringGlobe/assets/69278249/ea741c87-4a74-48e7-8b24-97498290ab72)